corehq/apps/locations/tests/test_location_fixtures.py

```diff
@@ -647,6 +652,42 @@ def test_include_only_location_types(self):
         ])
 
 
+@flag_enabled("RELATED_LOCATIONS")
+@mock.patch.object(Domain, 'uses_locations', lambda: True)  # removes dependency on accounting
+class RelatedLocationFixturesTest(LocationHierarchyTestCase, FixtureHasLocationsMixin):
+    """
+    - State
+        - County
+            - City
+        - Region
+            - Town
```

Comment:
Did you intend to add Region and Town to the tests?

Reply:
https://github.com/dimagi/commcare-hq/pull/21431/commits/09faad71a20c3eb6b31a380da7d7580de3d221c7